src/OpenSpaceToolkit/Astrodynamics/GuidanceLaw/SequentialGuidanceLaw.cpp

```diff
@@ -0,0 +1,104 @@
+/// Apache License 2.0
+
+#include <OpenSpaceToolkit/Core/Error.hpp>
+#include <OpenSpaceToolkit/Core/Type/Size.hpp>
+#include <OpenSpaceToolkit/Core/Utility.hpp>
+
+#include <OpenSpaceToolkit/Astrodynamics/GuidanceLaw/SequentialGuidanceLaw.hpp>
+
+namespace ostk
+{
+namespace astrodynamics
+{
+namespace guidancelaw
+{
+
+using ostk::core::container::Array;
+using ostk::core::container::Tuple;
+using ostk::core::type::Real;
+using ostk::core::type::Shared;
+using ostk::core::type::Size;
+
+using ostk::mathematics::object::Vector3d;
+
+using ostk::physics::coordinate::Frame;
+using ostk::physics::time::Instant;
+using ostk::physics::time::Interval;
+
+using ostk::astrodynamics::GuidanceLaw;
+
+SequentialGuidanceLaw::SequentialGuidanceLaw(
+    const Array<Tuple<Shared<GuidanceLaw>, Interval>>& aGuidanceLawWithIntervalArray
```

Comment:
Nit: Should this be a Pair instead of a Tuple to align with our pattern? Did you have a specific reason to make it a Tuple instead?